vite.config.ts

```diff
@@ -52,6 +75,98 @@ export default defineConfig(({ mode }) => {
       }),
       react(),
       svgr(),
+      VitePWA({
```

Comment:
Could baseUrl or a similar option partition the service worker caches for beta/v3 ?

Reply:
Maybe workbox.cacheId will work (main/beta or prod/staging)
https://developer.chrome.com/docs/workbox/modules/workbox-build#type-GeneratePartial

Reply:
Trying that now...

Reply:
This looks good. We had to explicitly name the non-pre-cache caches.